scripts/check-truffle-namespace-dependency-versions.js

```diff
@@ -67,7 +68,7 @@ function* checkDependencyVersionRange(candidate, dependency) {
 
   for (const depType of depTypes) {
     const deps = candidate[depType];
-    if (deps && deps[name]) {
+    if (typeof deps === "object" && deps[name]) {
```

Comment:
Note [according to specs](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#bundledependencies), `bundled?Dependencies` is an Array.

Reply:
Consistentcy is underrated, eh?

Reply:
Turns out it also doesn't include version info, so it's moot in this context (dropped it from the checks).